content/articles/securing-mongodb-in-linux/index.md

```diff
@@ -0,0 +1,230 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /securing-mongodb-in-linux/
+title: How to secure MongoDB in Linux
+description: This tutorial aims to show our readers how to protect their MongoDB setup by using some activation command lines in Linux.
+author: ali-benard
+date: 2022-01-27T00:00:00-17:00
+topics: [Security]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/securing-mongodb-in-linux/hero.jpg 
+    alt: Securing MongoDB in Linux Hero Image
+---
+### Content
+- [Prerequisites](#prerequisites)
+- [Introduction](#introduction)
+- [Check the environment](#check-the-environment)
+- [Securing MongoDB](#securing-mongodb)
+- [Create a user with system access](#create-a-user-with-system-access)
+- [Turn on authentication](#turn-on-authentication)
+- [Authentication Testing](#authentication-testing)
+- [Wrapping it up](#wrapping-it-up)
+
+### Prerequisites
+Before you start reading this tutorial you need to have the  following in your machine:
+- [Ubuntu](https://ubuntu.com/download/server) 20.05 server installed.
+- UFW-configured proxy server.
+- Installed [MongoDB](https://www.mongodb.com/try/download/community)
+
+### Introduction
+MongoDB is one of the most used modern web applications for a database. MongoDB is not secure by default. Once installed, one will have a difficult time before configuring it for authentication, since people can browse, write, delete, or modify data on your server without ever needing to log in or authenticate. Securing the database is simple and may be completed in a few steps. In this article, I'll show you how to protect your MongoDB setup by using some activation command lines.

```

Comment:
1. The third sentence in this paragraph is too long. Consider splitting it into two.
2. Avoid using `I` in "I'll show you how...". The focus should always be on the reader. Either use 'we' or 'you' .